integration-tests/tasks/rhtap-install.yaml

```diff
@@ -35,5 +45,37 @@ spec:
         git clone "$(params.git-url)" .
         git checkout "$(params.git-revision)"
 
+        # Export the environment variables from the secret
+        cat /workspace/config.env > config.sh
+        chmod +x config.sh
+        source ./config.sh
+
+        # Path to your values.yaml.tpl file
+        tpl_file="charts/values.yaml.tpl"
+
+        # Create the new integrations section
+        cat <<EOF >> "$tpl_file"
```

Comment:
Is this the supported way? I thought this should be done by calling various `rhtap-cli integration *` commands and there shouldn't be a reason to touch this template.

Reply:
Yeah... originally I didn't realize that. Thanks